internal/services/v1alpha1/schema_test.go

```diff
@@ -28,8 +28,27 @@ func TestSchemaReadNoPrefix(t *testing.T) {
 	ds, err := memdb.NewMemdbDatastore(0, 0, memdb.DisableGC, 0)
 	require.NoError(t, err)
 
-	srv := NewSchemaServer(ds, PrefixRequired)
-	_, err = srv.ReadSchema(context.Background(), &v1alpha1.ReadSchemaRequest{
+	// test relies on middleware
```

Comment:
There are far too many copies of code that looks just like this in this PR. Why?

Reply:
I was planning to use to buffconn PR to clean these up later, but I'll do it now so it's more reviewable.